doc/docs/cli-installation.md

```diff
@@ -27,16 +27,21 @@ After the setup, you can [start using Scala](https://docs.scala-lang.org/scala3/
 On Linux, download and run the coursier installer with
 
 ```bash
-$ curl -fL https://github.com/coursier/launchers/raw/master/cs-x86_64-pc-linux.gz | gzip -d > cs
+$ curl -fL "https://github.com/coursier/launchers/raw/master/cs-$(uname -m)-pc-linux.gz" | gzip -d > cs
 $ chmod +x cs
 $ ./cs setup
 ```
 
+Currently, we support the following architectures: x86, AMD, and ARM.
```

Comment:
Shouldn't that be ARM64 instead of ARM?

Reply:
I am pretty sure this should by ARM64, not just ARM. ARM is IMHO a way to broad definition. ARM64 whoever is a synonym for AARCH64, also see https://en.wikipedia.org/wiki/AArch64
And currently the binaries you offer are aarch64 only, so arm64 is accurate here.

Reply:
I agree with @mkurz. Just ARM would be 32 bit ARM (Raspberry Pi 1-3).

Reply:
Rather than `x86, AMD`, I'd just say `x86-64`. Here too, just `x86` means 32 bit processors.

Reply:
Indeed, thank you for the suggestions, this is fixed by 96ada0ba9

Reply:
BTW, why do we generate both `cs-amd64-pc-linux` and `cs-x86_64-pc-linux` if the underlying architecture is the same?